edtLHD/src/test/java/fr/univtln/lhd/facade/ScheduleTest.java

```diff
@@ -0,0 +1,38 @@
+package fr.univtln.lhd.facade;
+
+import fr.univtln.lhd.model.entities.dao.DAO;
+import fr.univtln.lhd.model.entities.dao.slots.SlotDAO;
+import fr.univtln.lhd.model.entities.dao.users.StudentDAO;
+import fr.univtln.lhd.model.entities.users.Student;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.threeten.extra.Interval;
+
+import java.sql.SQLException;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ScheduleTest {
+
+    @Test
+    //Kind of a test
+    void getSchedule() throws SQLException {
+        StudentDAO dao = StudentDAO.getInstance();
+        SlotDAO sdoa = SlotDAO.getInstance();
+        Interval timerange = sdoa.get(3).get().getTimeRange();
+        Student student = dao.get(1).get();
```

Comment:
Maybe this should be another PR. We can add tests later on.

Reply:
that's what I thought too with #66 , do you think we should delete this file ?

Reply:
Yes, I believe it's preferable